src/views/VideoView/VideoView.tsx

```diff
@@ -99,6 +102,23 @@ const VideoView: React.FC<RouteComponentProps> = () => {
             </>
           )}
         </DescriptionContainer>
+        <LicenseContainer>
+          {data?.video ? (
+            <>
+              <p>
+                License:{' '}
+                {data.video.license.type.__typename === 'KnownLicense' ? (
+                  <a href={data.video.license.type.url || ''}>{`${data.video.license.type.code}`}</a>
```

Comment:
The link should have `target="_blank"` and that should also imply `rel="noreferrer noopener"`

Reply:
Good point, changed that.